packages/components/psammead-media-player/src/Guidance/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { string, node } from 'prop-types';
+import styled from 'styled-components';
+import { C_WHITE } from '@bbc/psammead-styles/colours';
+import { mediaIcons } from '@bbc/psammead-assets/svgs';
+import { GEL_GROUP_2_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_LONG_PRIMER } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const GUIDANCE_BACKGROUND = 'rgba(34, 34, 34, 0.75)';
+
+const StyledGuidance = styled.div`
+  ${({ service }) => getSansRegular(service)}
+  ${GEL_LONG_PRIMER};
+`;
+
+const GuidanceWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  background-color: ${GUIDANCE_BACKGROUND};
+  color: ${C_WHITE};
+  padding: ${GEL_SPACING};
+`;
+
+const Content = styled.div`
+  display: flex;
+  align-items: center;
+  padding: ${GEL_SPACING_DBL} ${GEL_SPACING};
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING_DBL} ${GEL_SPACING};
+  }
+`;
+
+const GuidanceMessage = styled.strong`
+  padding: 0 0 0 ${GEL_SPACING};
```

Comment:
This should be `GEL_SPACING_HLF`, because the icon wrapper already includes a 4px margin.
Also, a suggestion, set the left padding independently with `padding-left: ${GEL_SPACING_HLF};`.